trade-finance-manager-ui/server/controllers/case/parties/index.test.js

```diff
@@ -125,6 +125,7 @@ describe('controllers - case - parties', () => {
           tfm: mockDeal.tfm,
           dealId: req.params._id,
           user: session.user,
+          urn: mockDeal.tfm.parties.exporter.partyUrn,
```

Comment:
This test is only concentrated at `exporter` party, other parties test should also be incorporated:

* Agent
* Indemnifier
* Bond Issuer
* Bond Beneficiary

Reply:
unable to add these but e2e tests cover these

Reply:
Sorry, can you please remind me again why are we unable to add these here?

Reply:
done